app/src/app/components/contents/runner/PacketRun.tsx

```diff
@@ -1,3 +1,43 @@
+import { Skeleton } from "../../Base/Skeleton";
+import { ErrorComponent } from "../common/ErrorComponent";
+import { useGetGitBranches } from "./hooks/useGetGitBranches";
+import { PacketRunForm } from "./run/PacketRunForm";
+
 export const PacketRun = () => {
-  return <div>Run</div>;
+  const { branchData, error, isLoading, mutate } = useGetGitBranches();
+
+  return (
+    <div>
+      <div>
+        <h2 className="text-2xl font-bold tracking-tight">Run</h2>
+        <p className="text-muted-foreground">Run a packet group to create packets</p>
```

Comment:
```suggestion
        <p className="text-muted-foreground">Run a packet group to create a new packet</p>
```

Maybe slightly more accurate - you'll only create one new packet when you run a packet group (once)!